IdentityCore/src/cache/crypto/MSIDAssymetricKeyPair+StkJwkExtensions.m

```diff
@@ -0,0 +1,151 @@
+// Copyright (c) Microsoft Corporation.
+// All rights reserved.
+//
+// This code is licensed under the MIT License.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files(the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions :
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+#import "MSIDAssymetricKeyPair+StkJwkExtensions.h"
+
+#import "MSIDJsonObject.h"
+#import "NSData+MSIDAssymetricKeyPairExtensions.h"
+
+@implementation MSIDAssymetricKeyPair (StkJwkExtensions)
```

Comment:
Is this iOS test failure likely related to my changes?

MSIDLastRequestTelemetryTests
  testUpdateTelemetryString_whenUpdatesFromDifferentThreads_shouldBeThreadSafe, ((telemetryObject.errorsInfo.count) equal to (4)) failed: ("0") is not equal to ("4")
  /Users/travis/build/AzureAD/microsoft-authentication-library-common-for-objc/IdentityCore/tests/MSIDLastRequestTelemetryTests.m:88
  ```
    
    XCTAssertEqual(telemetryObject.errorsInfo.count, 4);
}
  ```